packages/listbox/types/ListboxMixinTypes.d.ts

```diff
@@ -41,10 +41,17 @@ export declare class ListboxHost {
 
   public setCheckedIndex(index: number): void;
 
+  /** Reset interaction states and modelValue */
+  public reset(): void;
+
   protected _scrollTargetNode: LionOptions;
 
   protected _listboxNode: LionOptions;
 
+  protected _listboxReceivesNoFocus: Boolean;
```

Comment:
not blocking but we tend to use lowercase for primitives. For typescript I think there's actually a significant difference, for JSDocs it doesn't matter

Reply:
makes sense. Will adapt in the other mr 👍